client/src/components/Users/HostCard.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Button, Card, ListGroup } from 'react-bootstrap';
+import { run as runHolder } from 'holderjs/holder';
+import { useEffect } from 'react';
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import { Link } from 'react-router-dom';
+import moment from 'moment';
```

Comment:
We are not using moment. It is just a leftover from the boilerplate. Please remove it in the whole code and in the package.json and replace with dayjs